### PR TITLE
pause community meeting for a little while

### PR DESCRIPTION
**Pull request by cfryanr**
We've decided to pause our biweekly community meetings for a little while. We'll resume them at a later date. In the meantime, we will likely schedule ad hoc public meetings for things like release announcements and community feedback gathering. Please subscribe to our mailing list and Slack channel for meeting updates.

For now, update the web site to remove the schedule details of the biweekly community meetings.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by cfryanr** [comment]
@jonasrosland @anjaltelang PR for your review.

Deploy previews:
- Web site: https://deploy-preview-1240--pinniped.netlify.app
- CONTRIBUTING.md: https://github.com/vmware-tanzu/pinniped/blob/0a235b3d955ce53ca06487d172c1f55711d11628/CONTRIBUTING.md
- README.md: https://github.com/vmware-tanzu/pinniped/blob/0a235b3d955ce53ca06487d172c1f55711d11628/README.md
- ROADMAP.md: https://github.com/vmware-tanzu/pinniped/blob/0a235b3d955ce53ca06487d172c1f55711d11628/ROADMAP.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,18 +17,6 @@ See [SCOPE.md](./SCOPE.md) for some guidelines about what we consider in and out
...
 
-## Community Meetings
-
-Pinniped is better because of our contributors and [maintainers](MAINTAINERS.md). It is because of you that we can bring great
-software to the community. Please join us during our online community meetings,
-occurring every first and third Thursday of the month at 9 AM PT / 12 PM ET.
-Use [this Zoom Link](https://go.pinniped.dev/community/zoom)
-to attend and add any agenda items you wish to discuss
-to [the notes document](https://hackmd.io/rd_kVJhjQfOvfAWzK8A3tQ?view).
-Join our [Google Group](https://groups.google.com/g/project-pinniped) to receive invites to this meeting.
```
Comment (jonasrosland) [inline CONTRIBUTING.md:25]: I would keep a variation of this, like "Join our Google Group for invites to detailed release meetings, updates the project, and related conversations.

Comment (jonasrosland) [reply CONTRIBUTING.md:25]: This would be across all the docs where this was changed.

File: site/content/community/_index.html [hunk 2]
```diff
@@ -35,11 +35,11 @@ <h3><a href="{{< param "slack_url" >}}">Chat with us on our Slack channel</a></h
...
-                <p>Please join us during our online <a href="https://go.pinniped.dev/community/agenda">Community Meetings</a>, occurring every first and third Thursday of the month at 9 AM PT / 12 PM ET.</p>
-                <p>Use <a href="https://go.pinniped.dev/community/zoom">this Zoom link</a> to attend and add any agenda items you wish to discuss to <a href="https://go.pinniped.dev/community/agenda">the notes document</a>.</p>
-                <p>Join our <a href="https://groups.google.com/u/1/g/project-pinniped">Google Group</a> to receive invites to the meeting</p>
+<!--                <p>Please join us during our online <a href="https://go.pinniped.dev/community/agenda">Community Meetings</a>, occurring every first and third Thursday of the month at 9 AM PT / 12 PM ET.</p>-->
+<!--                <p>Use <a href="https://go.pinniped.dev/community/zoom">this Zoom link</a> to attend and add any agenda items you wish to discuss to <a href="https://go.pinniped.dev/community/agenda">the notes document</a>.</p>-->
+                <p>Join our <a href="https://groups.google.com/u/1/g/project-pinniped">Google Group</a> to receive invites to community meetings</p>
```
Comment (jonasrosland) [inline site/content/community/_index.html:40]: Add the same blurb about joining the group for updates, release meetings etc here.

**Review by jonasrosland — CHANGES_REQUESTED** [review]
Just minor changes requested, otherwise this looks good!

File: CONTRIBUTING.md (hunk 1 above)
Comment (cfryanr) [reply CONTRIBUTING.md:25]: Thanks, updated the original commit on this PR to include this.

File: site/content/community/_index.html (hunk 2 above)
Comment (cfryanr) [reply site/content/community/_index.html]: Thanks, updated the original commit on this PR to include this.

**Pull request merged**